_posts/2019-11-12-docker-rename-to-container.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Pulp Docker plugin will be renamed to Pulp Container plugin in Pulp3
+author: Ina Panova
+tags:
+  - docker
+  - container
+---
+
+With Pulp 3 GA pulp_container 1.0 will come along.
+
+## Why renaming pulp_docker to pulp_container in Pulp3?
+
+* pulp_docker plugin implies management of docker format images only. This plugin will stay in Pulp 2.
+* pulp_container won't be tied to docker format images, this will enable us to add support for OCI format in the future.
+
+## What does this mean for me?
+
+Pulp 2.y users who manage Docker content should be able to migrate their production
+systems to Pulp 3.0 with the Pulp Container plugin installed.
+Pulp 3 Container plugin has full feature parity with Pulp 2 Docker plugin.
+
+Pulp 3 users will need to uninstall pulp_docker plugin and install pulp_container one.
```

Comment:
Pulp 3 users will need to uninstall the pulp_docker plugin and install the pulp_container one.